.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+### Description
+
+<!-- Describe the purpose of this PR, what being adding and/or fixed -->
+
+### Notes to the reviewers
+
+<!-- In this section you can include notes directed to the reviewers, like explaining why some parts
+of the PR were done in a specific way -->
+
+### Checklists
+
+#### All Submissions:
+
+* [] I've signed all my commits
+* [] I followed the [contribution guidelines](https://github.com/bitcoindevkit/bdk/blob/master/CONTRIBUTING.md)
+* [] I ran `cargo fmt` and `cargo clippy` before committing
+
+#### New Features:
+
+* [] I've added tests for the new feature
+* [] I've added docs for the new feature
+* [] I've updated `CHANGELOG.md`
+
+#### Bugfixes:
+
+* [] This pull request breaks the existing API
+* [] I've added a test to reproduce the issue which are now passing
```

Comment:
small thing, to make plural agree.. `I've added tests to reproduce the issue which are now passing`